pyaedt/maxwell.py

```diff
@@ -1982,7 +1982,8 @@ def create_external_circuit(self, circuit_design=None):
 
         circuit = MaxwellCircuit(design=circuit_design)
 
-        wdgs = self.excitations_by_type["Winding Group"]
+        wdgs = self.excitations_by_type["Winding"]
```

Comment:
@IreneWoyna you should check both cases :)

Reply:
see my comment below, Maxwell throws an error already when I try to check the "winding group" and doesn't create the circuit...

Reply:
@IreneWoyna yes but you should check that "Winding" or "Winding Group" are keys of the excitations_by_type dictionary. If yes you get you excitations as: self.excitation_by_type[excitation_type].

Reply:
@gmalinve now it should work